extensions/ql-vscode/src/remote-queries/analyses-results-manager.ts

```diff
@@ -15,6 +15,7 @@ export class AnalysesResultsManager {
 
   constructor(
     private readonly ctx: ExtensionContext,
+    readonly storagePath: string,
```

Comment:
I don't think this is being used anywhere anymore so can be reverted.

Reply:
It is being used.